### PR TITLE
New togglebutton for Modes 

### PR DESCRIPTION
**Pull request by signedav**
So you not have to go over the menu every time.

This PR replaces the #341 PR.

Looks like this:
toggle2

Additionally the GPS button is greyed out when inactive.

### CODE REVIEW

**Comment by qfield-fairy** [comment]
Uploaded test apks for [armv7](https://opengis.ch/download/qfield/ci-builds/qfield-dev-344-598b93dd95574e49001b375fdc21a95b3ab1f87e-armv7.apk) and [x86](https://opengis.ch/download/qfield/ci-builds/qfield-dev-344-598b93dd95574e49001b375fdc21a95b3ab1f87e-x86.apk)

**Comment by giedrioks26** [comment]
works perfectly, but the "can't open database: ..." problem persists. Also wrong GPS location

**Comment by signedav** [comment]
Yes. Thanks. We are on the other problems still.

**Comment by giedrioks26** [comment]
@signedav what do you think about it #97, #96 ?

**Comment by signedav** [comment]
Both are really nice enhancements @giedrioks26. But we have to discuss the time resources we have at the moment.

**Comment by signedav** [comment]
I would really like to push it, because to switch modes over the menu is annoying.
@3nids you told me, something with the margins is not okay. Possibly we can continue the discussion here.
> it feels like not being in perfect harmony with the whole UX.

Do you mean the handling or the layout? I have these other examples here:
toggle_2_grey_border
toggle_3_noframe
toggle_1_dave
toggle_2_denis

**Comment by qfield-fairy** [comment]
Uploaded test apks for [armv7](https://opengis.ch/download/qfield/ci-builds/qfield-dev-344-af0dca2e88ae698f5dd7d458dc58e13846c15740-armv7.apk) and [x86](https://opengis.ch/download/qfield/ci-builds/qfield-dev-344-af0dca2e88ae698f5dd7d458dc58e13846c15740-x86.apk)

**Comment by signedav** [comment]
screenshot_2018-10-03-15-43-53
Forgot to make it green instead of grey. Will do it. Probably you have already an opinion for it @3nids @mbernasocchi @m-kuhn

**Comment by signedav** [comment]
(actually I really liked the navigation compass symbol :slightly_smiling_face: )

**Comment by qfield-fairy** [comment]
Uploaded test apks for [armv7](https://opengis.ch/download/qfield/ci-builds/qfield-dev-344-2c7ea7d1376764fe95bd973039774c3ca5dd9eff-armv7.apk) and [x86](https://opengis.ch/download/qfield/ci-builds/qfield-dev-344-2c7ea7d1376764fe95bd973039774c3ca5dd9eff-x86.apk)

File: src/qml/DashBoard.qml [hunk 1]
```diff
@@ -66,6 +67,64 @@ Controls.Pane {
 
           onClicked: showMenu()
         }
+
+        Rectangle {
+          //empty space
+          height: 48 * dp
+          width: 48 * dp
```
Comment (3nids) [inline src/qml/DashBoard.qml]: go for 56 here, both width and height.

Comment (3nids) [reply src/qml/DashBoard.qml]: and adapt children elements

**Comment by 3nids** [comment]
yes, I prefer the compass too.

what about map icon....that would make sense ;)
image
https://material.io/tools/icons/?icon=map&style=baseline

**Comment by signedav** [comment]
So the map icon. No information-i and no compass. 
Is this fine for everyone? Speak now or forever hold your peace :wink: 
@mbernasocchi @marioba @3nids @m-kuhn

**Comment by tdunbar12** [comment]
I'd like an I option, it's just such a handy way to investigate a single
line in the database.

Tom

**Comment by 3nids** [comment]
@tdunbar12 it would actually not really be an identify tool like in QGIS but rather a browsing map mode (where identifiying is of course possible on the map).

@signedav my vote to map icon

**Comment by mbernasocchi** [comment]
@signedav do you have an image with th map? I think I tend for the map

**Comment by signedav** [comment]
toggletoggletoggle
toggle toggle toggle toggle

**Comment by signedav** [comment]
@3nids not sure about the size. What size (visually) you want to have the boxes / images in the end?

**Comment by mbernasocchi** [comment]
perfect

**Comment by 3nids** [comment]
awesome, looks good as it!

**Comment by signedav** [comment]
Thanks guys!

I've just seen that the compass and the information icons are still added in this PR - should I delete them again in another PR or should we keep them in the app to have them accessible when we need them.

**Comment by 3nids** [comment]
no big deal to me...I tend to prefer the clean approach (strictly what we use) but on the other hand playing with the icons is a PITA....

**Comment by mbernasocchi** [comment]
leave them there, they are ready and we might need them eventually.

**Comment by signedav** [comment]
Okay, thanks.

**Pull request merged**